noir-projects/noir-contracts/contracts/token_contract/src/test/refunds.nr

```diff
@@ -65,12 +65,10 @@ unconstrained fn setup_refund_success() {
     utils::check_private_balance(token_contract_address, fee_payer, 1)
 }
 
-// TODO(#7694): Ideally we would check the error message here but it's currently not supported by TXE. Once this
-// is supported, check the message here and delete try deleting the corresponding e2e test.
-// #[test(should_fail_with = "funded amount not enough to cover tx fee")]
-#[test(should_fail)]
+//#[test(should_fail_with="funded amount not enough to cover tx fee")]
+#[test(should_fail_with = "Balance too low")]
```

Comment:
Rethrows are fixed in TXE just like in PXE (hackily, but we get proper error messages...will review with @fcarreiro when rethrows in the AVM work). 

This test is essentially broken and I left the comment to remind ourselves of the actual error message it should be throwing. I didn't want to remove it since we are reworking how the partial note stuff works, so planned I to bother @benesjan to fix it when he finishes.

Reply:
Can we add a small comment explaining that the commented out message is the one we should receive?